app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
+    page = IntegerField(default=1, validators=(NumberRange(min=1),))
+    status = SelectMultipleField("Status", choices=(
+        ("live", "Live",),
+        ("closed", "Closed",)
+    ))
+    # lot choices expected to be set at runtime
+    lot = SelectMultipleField("Category")
+
+    def __init__(self, *args, **kwargs):
+        """
+            Accepts extra keyword arguments:
```

Comment:
Should this be `requires` rather than `accepts`?  Neither of these args is optional.